diff --git a/src/common/common_w32res.rc b/src/common/common_w32res.rc
index d05d22e..a70de34 100644
--- a/src/common/common_w32res.rc
+++ b/src/common/common_w32res.rc
@@ -6,9 +6,7 @@
  */
 
 #include <winresrc.h>
-#ifdef HAVE_CONFIG_H
-# include "config.h"
-#endif
+#include "config.h"
 #define LZMA_H_INTERNAL
 #define LZMA_H_INTERNAL_RC
 #include "lzma/version.h"
diff --git a/src/common/sysdefs.h b/src/common/sysdefs.h
index df7ecf4..c596cae 100644
--- a/src/common/sysdefs.h
+++ b/src/common/sysdefs.h
@@ -20,9 +20,7 @@
 // Includes //
 //////////////
 
-#ifdef HAVE_CONFIG_H
-#	include <config.h>
-#endif
+#include <config.h>
 
 // Get standard-compliant stdio functions under MinGW and MinGW-w64.
 #ifdef __MINGW32__
diff --git a/src/common/tuklib_config.h b/src/common/tuklib_config.h
index 549cb24..3fe2145 100644
--- a/src/common/tuklib_config.h
+++ b/src/common/tuklib_config.h
@@ -1,7 +1 @@
-#ifdef HAVE_CONFIG_H
-#	include "sysdefs.h"
-#else
-#	include <stddef.h>
-#	include <inttypes.h>
-#	include <limits.h>
-#endif
+#include "sysdefs.h"
